react-components/src/data/details/BasicDetails.js

```diff
@@ -52,7 +51,7 @@ function Md5(props) {
             <DEHyperlink text={getMessage("view")}/>
         </td>);
     } else {
-        return <td id={props.id} className={css(styles.value)}>{getMessage("folders")}</td>;
+        return <td id={props.id} className={props.classes.detailsValue}>{getMessage("emptyValue")}</td>;
```

Comment:
Did `getMessage("folders")` intentionally change to `getMessage("emptyValue")` here?

Reply:
It was a bug. So if MD5 is null or empty I should display empty value but it was displaying `folders` 🤦‍♂️